src/event.py

```diff
@@ -1,4 +1,205 @@
+from random import sample
+
+
 class Event:
-    def __init__(self, type, time):
-        self.type = type
+    def __init__(self, time):
         self.time = time
+
+    def print_message(self, messages, params):
```

Comment:
It seems that the **message** parameter can be a field of the _Event_ class - all its subclasses have it.